app/models/firm.rb

```diff
@@ -173,6 +174,10 @@ def primary_advice_method
     infer_primary_advice_method
   end
 
+  def main_office
+    offices.first
```

Comment:
The rule I'm following is: the offices are sorted by `created_at`. The first office created is considered the "main office". This means we can treat that record as special, e.g. prevent it being deleted through the UI and so on.